meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-updateContainer.sh

```
@@ -1,80 +1,123 @@
-       date >> /tmp/updateContainer/MANUAL_UPDATE
-
-       rm -rf /tmp/updateContainer/globalping-probe.frozen/
-       umount /tmp/updateContainer
 
-       sleep 5
+       echo "Starting container update process" > /dev/tty4
 
-       reboot
-       echo "1" > /dev/watchdog
-       while :; do  sleep 2; done
+       docker stop $(docker ps -a -q)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)